widgets/waterTracker/index.html

```diff
@@ -29,7 +31,7 @@ <h1>Water Tracker</h1>
         <div>
             <div id="limit"></div>
             <input id="goalWater" type="number" placeholder="300" min="10" max="10000"/>
-            <button class="btn" id="submit">Set your limit</button>
+            <button class="btn" id="submit">Set your goal</button>
```

Comment:
I'm glad you made this change from 'limit' to 'goal'.  It makes more sense to me.